tools/go-cli/go-whisk-cli/commands/package.go

```diff
@@ -598,19 +550,19 @@ func init() {
     packageCreateCmd.Flags().StringSliceVarP(&flags.common.annotation, "annotation", "a", []string{}, "annotations")
     packageCreateCmd.Flags().StringSliceVarP(&flags.common.param, "param", "p", []string{}, "default parameters")
     packageCreateCmd.Flags().StringVarP(&flags.xPackage.serviceGUID, "service_guid", "s", "", "a unique identifier of the service")
-    packageCreateCmd.Flags().StringVar(&flags.common.shared, "shared", "" , "shared action (default: private)")
+    packageCreateCmd.Flags().BoolVar(&flags.common.shared, "shared", false, "make a package publically accessible")
```

Comment:
"make a package.." -> "make package...(default: private)" (remove the 'a' indefinite article).  Same comment for other `shared` help text